leaflet/index.d.ts

```diff
@@ -1524,6 +1524,26 @@ declare namespace L {
         export const svg: boolean;
     }
 
+    export namespace Util {
+        export function extend(dest: {}, src?: {}): {};
+        export function create(proto: {}, properties?: {}): {};
+        export function bind(fn: () => void, ...obj: Array<{}>): () => void;
+        export function stamp(obj: {}): number;
+        export function throttle(fn: () => void, time: number, context: {}): () => void;
+        export function wrapNum(num: number, range: number[], includeMax?: boolean): number;
+        export function falseFn(): () => void;
+        export function formatNum(num: number, digits?: number): number;
+        export function trim(str: string): string;
+        export function splitWords(str: string): string[];
+        export function setOptions(obj: {}, options: {}): {};
+        export function getParamString(obj: {}, existingUrl?: string, uppercase?: boolean): string;
+        export function template(str: string, data: {}): string;
+        export function isArray(obj: {}): boolean;
+        export function indexOf(array: any[], el: {}): number;
+        export function requestAnimFrame(fn: () => void, context?: {}, immediate?: boolean): number;
+        export function cancelAnimFrame(id: number): undefined;
+        export const emptyImageUrl: string;
+    }
```

Comment:
Doesn't look like `lastId` is exported or modified when using `stamp`.

https://github.com/Leaflet/Leaflet/blob/master/src/core/Util.js#L48-L57

Going to include it and make a PR to leaflet to export that variable